gui/src/callbacks.cc

```diff
@@ -443,10 +443,15 @@ void on_scroom_bootstrap(const FileNameMap& newFilenames)
   }
 
 
-  aboutDialogXml = glade_xml_new(xmlFileName.c_str(), "aboutDialog", nullptr);
+  aboutDialogXml = gtk_builder_new();
+  gchar** obj    = new gchar*[1];
+  obj[0]         = "aboutDialog";
+  gtk_builder_add_objects_from_file(aboutDialogXml, xmlFileName.c_str(), obj, NULL);
```

Comment:
Could you have a look at memory management? I think we're leaking the GtkBuilder we're creating here, as well as the `obj` array we allocated.

The `obj` array should be null-terminated, i.e. `obj[1]` should be `NULL`. `gtk_builder_add_objects_from_file()` will look at `obj[i]` for ever larger values of `i`, and try to construct those, until it encounters an `obj[i]` that equals `NULL`.

This might very well be the cause of the crash your're seeing.

My recommendation would be to make `obj` a `std::vector<gchar>`, and then `push_back()` both `"aboutDialog"` and `nullptr`, and then pass `obj.data()` to `gtk_builder_add_objects_from_file()`.

Also, I think the call to `gtk_builder_add_object_from_file()` should perhaps be postponed until after we check `aboutDialogXml` for null, further down (although the api documentation suggests that `gtk_builder_new()` will never return null)

Lastly, the documentation for `gtk_builder_new()` suggest we should be using `gtk_builder_new_from_file()` instead?